lib/middleware/app_middleware.dart

```diff
@@ -98,10 +98,13 @@ class PlumbDatabaseStreamMiddleware
           databaseService.storeStream.listen(
             store.dispatch,
             onError: (dynamic error, StackTrace trace) => store.dispatch(
-              AddProblem.from(
-                  message: error.toString(),
-                  traceString: trace.toString(),
-                  type: ProblemType.databaseStoreController),
+              AddProblem(
+                problem: DatabaseStoreControllerProblem.by(
```

Comment:
Yeah this is weird one to name. My instinct would be to use the action name and make it "PlumbDatabaseStreamProblem" as it doesn't seem any less clear than "DatabaseStoreControllerProblem" but does give an indication of the action that resulted in the problem. What do you think?

Reply:
I find that to make a lot of sense, I'll revise it in the next commit 👍